misc/bstr.h

```diff
@@ -56,11 +56,11 @@ static inline struct bstr bstrdup(void *talloc_ctx, struct bstr str)
     return r;
 }
 
-#define bstr0_s(s) (struct bstr){(unsigned char *)(s), (s) ? strlen(s) : 0}
+#define bstr0_lit(s) {(unsigned char *)(s), sizeof("" s) - 1}
```

Comment:
the choice not to include `(struct bstr)` here seems weird

Reply:
It wouldn't be a constant expression anymore if you make it a compound literal. Which is pretty stupid, but it is what it is.

Reply:
> Which is pretty stupid, but it is what it is.

I don't like it either, but this is what we get for using C. mpv in Zig when?